linkml_runtime/utils/schemaview.py

```diff
@@ -48,31 +48,33 @@
     from collections.abc import Mapping
     from types import NotImplementedType
 
-    from linkml_runtime.utils.metamodelcore import URIorCURIE
+    from linkml_runtime.utils.metamodelcore import URI, URIorCURIE
 
 
 logger = logging.getLogger(__name__)
 
 MAPPING_TYPE = str  ## e.g. broad, exact, related, ...
 CACHE_SIZE = 1024
 
-SLOTS = "slots"
```

Comment:
You are not moving, it but removing it, right? But it's being used, so it'll fail [here](https://github.com/linkml/linkml-runtime/pull/419/commits/3c38b2e4a27367f078d8ccff10276c4fe6862821#diff-a61f79c84f3e6ee5a5296f31b53d3f01d5287589c69f8dddd96be19113a6eeceR521) and in the newly defined `ELEMENTS` constant.